docs/users/cartography.rst

```diff
@@ -269,26 +269,36 @@ Using the toolbar, the user can:
 * Edit ( |ToolEdit| ) the selected symbolizer / rule,
 
 
-.. |ToolAdd| image:: ../_images/thematic_tool_add.png
+.. |ToolAdd| image:: ../_images/cartography/thematic_tool_add.png
               :alt: Add icon
               :width: 16 pt
 
-.. |ToolRemove| image:: ../_images/thematic_tool_remove.png
+.. |ToolRemove| image:: ../_images/cartography/thematic_tool_remove.png
               :alt: Remove icon
               :width: 16 pt
 
-.. |ToolEdit| image:: ../_images/thematic_tool_edit.png
+.. |ToolEdit| image:: ../_images/cartography/thematic_tool_edit.png
               :alt: Edit icon
               :width: 16 pt
 
-.. |ToolUp| image:: ../_images/thematic_tool_up.png
+.. |ToolUp| image:: ../_images/cartography/thematic_tool_up.png
               :alt: Up icon
               :width: 16 pt
 
-.. |ToolDown| image:: ../_images/thematic_tool_down.png
+.. |ToolDown| image:: ../_images/cartography/thematic_tool_down.png
               :alt: Down icon
               :width: 16 pt
 
 
+Compose & export maps
+-------------------------------------------
+
+With the `Map Composer`_ plugin, user can compose maps. This interface gives users the opportunity to add text, orientation, scale, \... and to export the result in .pdf or image files. A detailled documentation is provided `HERE`_.
```

Comment:
A map layout is a collection of map elements organized on a page and designed for map printing. With The Map Composer you can add map elements such  text, orientation, scale. Once you have completed a map layout, you can print it or export it  of output formats as   .pdf or image file